src/DependencyInjection/Configuration.php

```diff
@@ -37,8 +39,15 @@ private function addServersSection($rootNode)
                     ->useAttributeAsKey('alias', false)
                     ->prototype('array')
                         ->children()
-                            ->scalarNode('ip')->defaultValue('localhost')->end()
+                            ->scalarNode('host')->defaultValue('localhost')->end()
                             ->scalarNode('port')->defaultValue(6379)->end()
+                            ->integerNode('database')->defaultValue(0)->end()
+                            ->scalarNode('scheme')->defaultValue('tcp')->end()
+                            ->booleanNode('async_connect')->defaultValue('false')->end()
+                            ->booleanNode('persistent')->defaultValue('false')->end()
+                            ->floatNode('timeout')->defaultValue(10)->min(0)->end()
+                            ->floatNode('read_write_timeout')->min(0)->end()
+                            ->integerNode('reconnect')->defaultValue(0)->min(0)->max(5)
```

Comment:
Why `max: 5`, is there a major problem with an higher value?

Reply:
Its just the time it takes, nb reconnect * timeout. But i will remove the maximum value